queries.sql

```diff
@@ -107,6 +107,38 @@ FROM animals
 WHERE birth_year BETWEEN 1990 AND 2000
 GROUP BY species;
 
+SELECT animals.name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
```

Comment:
- [ Required ] I think it would be nice if you add a query that answers the following question _List of all animals that are Pokemon (their type is Pokemon)._ I could not find a query that answers this question in this file, yet it's part of the project requirements.

## Project requirements
![Screenshot from 2023-11-10 19-18-38](https://github.com/Surafels/Vet-clinic-database/assets/16392046/216ef76f-8157-485e-8e9d-6a021e1b87da)